### PR TITLE
Added first test

### PR DESCRIPTION
**Pull request by NateUniAccount**

### CODE REVIEW

File: src/evolv/io/tests/NameGeneratorTest.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import evolv.io.NameGenerator;
+
+public class NameGeneratorTest {
+	@Test
+	public void testNewName() {
+		for (int i = 0; i < 1_000; i++) {
```
Comment (PRabahy) [inline src/evolv/io/tests/NameGeneratorTest.java]: I've never seen that way of writing 1000 before. Does that actually work?

Comment (PRabahy) [inline src/evolv/io/tests/NameGeneratorTest.java]: Well, I guess I just learned something new. https://docs.oracle.com/javase/7/docs/technotes/guides/language/underscores-literals.html

File: pom.xml [hunk 2]
```diff
@@ -22,5 +22,10 @@
   		<artifactId>core</artifactId>
   		<version>3.2.3</version>
   	</dependency>
+    <dependency>
```
Comment (PRabahy) [inline pom.xml]: Just trivia, but can you make the indentation of <dependency> consistent?

File: src/evolv/io/tests/NameGeneratorTest.java (hunk 1 above)
Comment (NateUniAccount) [inline src/evolv/io/tests/NameGeneratorTest.java]: It's pretty cool huh, makes it a lot easier to see what number it is

File: pom.xml (hunk 2 above)
Comment (NateUniAccount) [inline pom.xml]: sure yeah will do

**Pull request merged**